client/extensions/woocommerce/state/ui/README.md

```diff
@@ -0,0 +1,20 @@
+State
+=====
+
+This directory contains the UI behavior describing the application state for the
+woocommerce extension. This is a sub-tree of the woocommerce state, stored in
+`state.extensions.woocommerce.ui`.
+
+## Structure
+
+### [`payment`](ui/README.md)
```

Comment:
If you're following the `state/sites` readme, this should link off to sub-tree readmes — so there should be a readme in `ui/payment`, and that should follow the format of, for example, [the settings general readme](https://github.com/Automattic/wp-calypso/blob/master/client/extensions/woocommerce/state/sites/settings/general/README.md)